zeppelin-zengine/src/main/java/org/apache/zeppelin/conf/ZeppelinConfiguration.java

```diff
@@ -503,6 +503,7 @@ public String getWebsocketMaxTextMessageSize() {
     // Decide when new note is created, interpreter settings will be binded automatically or not.
     ZEPPELIN_NOTEBOOK_AUTO_INTERPRETER_BINDING("zeppelin.notebook.autoInterpreterBinding", true),
     ZEPPELIN_CONF_DIR("zeppelin.conf.dir", "conf"),
+    ZEPPELIN_INTERPRETER_DOWNLOAD_DIR("zeppelin.interpreter.download.dir", "interpreter"),
```

Comment:
nit: this line might better be placed next to `ZEPPELIN_INTERPRETER_DIR("zeppelin.interpreter.dir", "interpreter")`.

Could you also add new configuration to [docs/install/install.md](https://github.com/apache/incubator-zeppelin/blob/master/docs/install/install.md#zeppelin-configuration) and [conf/zeppelin-site.xml.template](https://github.com/apache/incubator-zeppelin/blob/master/conf/zeppelin-site.xml.template#L172)